src/pages/Experiment.less

```diff
@@ -0,0 +1,44 @@
+.ex-page {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+
+    .navbar {
+        flex-shrink: 0;
+        flex-grow: 0;
+    }
+
+    &__loader {
+        margin-top: 40px;
+        text-align: center;
+    }
+
+    &__main {
+        flex-grow: 1;
+        display: flex;
+        flex-direction: column;
+    }
+
+    &__progress {
+        margin-bottom: 20px;
```

Comment:
If we could rely on Bootstrap grid system, we could get rid of hardcoded spacings